### PR TITLE
Add MP4K 3 tournament page

### PR DESCRIPTION
**Pull request by WalterToro**

### CODE REVIEW

File: wiki/Tournaments/MP4K/3/en.md [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+
+- If the beatmap ends in a draw, the game will be nullified and the beatmap will be replayed.
+
+#### Disconnections
+
+- Maps with a player disconnection that occurs within the first 30 seconds, or 25% of the song (whichever comes first) will be nullified and re-attempted.
+
+- Matches will have a two disconnection limit -- upon the third disconnection, a point will be handed immediately to the player still active within the lobby.
+
+- A two minute timer will be placed on a disconnected player when the song played has concluded. When this timer hits 0, the grace period mentioned in **Protocol** will be (re)activated. If the grace period timer hits 0, the match is forfeited, and the other remaining player will be declared the winner.
```
Comment (cl8n) [inline wiki/Tournaments/MP4K/3/en.md]:
```suggestion
- A two minute timer will be placed on a disconnected player when the song played has concluded. When this timer hits 0, the grace period mentioned in [Protocol](#protocol) will be (re)activated. If the grace period timer hits 0, the match is forfeited, and the other remaining player will be declared the winner.
```

File: wiki/Tournaments/MP4K/3/en.md [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+
+- In a loser's bracket match, the player who meets the winning condition in a match will be declared the winner of the match and will move further into the bracket. The loser of the match will be eliminated from the tournament.
+
+- Players who are in the winner's bracket are required to play in one match in each stage. The only exception to this is if the player is the sole player in winner bracket in the Grand Finals stage, where the player is required to play in up to two matches if the player were to lose his first match in the Grand Finals stage.
+
+- Players who are in the loser's bracket at the start of Finals Week 1 or further are required to play in two matches in a single stage. However, players who lose their first match in the losers bracket in those stages will no longer need to play in any further matches in the tournament, for they are eliminated from the tournament.
+
+### Tournament Play
+
+#### Protocol
```
Comment (cl8n) [inline wiki/Tournaments/MP4K/3/en.md]:
```suggestion
#### <a id="protocol"></a>Protocol
```
for the other link to work cuz this is a level > 3 heading

File: wiki/Tournaments/MP4K/3/en.md [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+| :-- | :-- |
+| Host | ![][flag_PL] [Kamikaze](https://osu.ppy.sh/users/2124783) |
+| Coordinator | ![][flag_PL] [Marleza](https://osu.ppy.sh/users/5396181), ![][flag_PL] [Open](https://osu.ppy.sh/users/7082921) |
+| Map Selectors | ![][flag_MY] [cheewee10](https://osu.ppy.sh/users/4477497), ![][flag_PL] [Kamikaze](https://osu.ppy.sh/users/2124783), ![][flag_ID] [-Rem-](https://osu.ppy.sh/users/5767941), ![][flag_CL] [WalterToro](https://osu.ppy.sh/users/5281416) |
+| Referees | ![][flag_RU] [AnatOWJIya](https://osu.ppy.sh/users/486264), ![][flag_PL] [Arkitev](https://osu.ppy.sh/users/6905790), ![][flag_PL] [Benzopirene](https://osu.ppy.sh/users/1887068), ![][flag_PL] [Cyan\_Star](https://osu.ppy.sh/users/8679184), ![][flag_PL] [Kamikaze](https://osu.ppy.sh/users/2124783), ![][flag_PL] [LaviSorrow](https://osu.ppy.sh/users/9966768), ![][flag_PL] [Moskas](https://osu.ppy.sh/users/1934077), ![][flag_PL] [MrDorian](https://osu.ppy.sh/users/1559983), ![][flag_PL] [m666](https://osu.ppy.sh/users/8623171), ![][flag_PL] [Open](https://osu.ppy.sh/users/7082921), ![][flag_PL] [SitekX](https://osu.ppy.sh/users/3840946), ![][flag_CL] [WalterToro](https://osu.ppy.sh/users/5281416) |
```
Comment (cl8n) [inline wiki/Tournaments/MP4K/3/en.md]:
```suggestion
| Referees | ![][flag_RU] [AnatOWJIya](https://osu.ppy.sh/users/486264), ![][flag_PL] [Arkitev](https://osu.ppy.sh/users/6905790), ![][flag_PL] [Benzopirene](https://osu.ppy.sh/users/1887068), ![][flag_PL] [Cyan_Star](https://osu.ppy.sh/users/8679184), ![][flag_PL] [Kamikaze](https://osu.ppy.sh/users/2124783), ![][flag_PL] [LaviSorrow](https://osu.ppy.sh/users/9966768), ![][flag_PL] [m666](https://osu.ppy.sh/users/8623171), ![][flag_PL] [Moskas](https://osu.ppy.sh/users/1934077), ![][flag_PL] [MrDorian](https://osu.ppy.sh/users/1559983), ![][flag_PL] [Open](https://osu.ppy.sh/users/7082921), ![][flag_PL] [SitekX](https://osu.ppy.sh/users/3840946), ![][flag_CL] [WalterToro](https://osu.ppy.sh/users/5281416) |
```

File: wiki/Tournaments/MP4K/3/en.md [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | 80 PLN and 2 months of osu!supporter |
+
+![MP4K 3 Badge](img/badge.png "MP4K 3 Winner Badge")
+
+## Organisation
+
+| Position | Member(s) |
+| :-- | :-- |
+| Host | ![][flag_PL] [Kamikaze](https://osu.ppy.sh/users/2124783) |
+| Coordinator | ![][flag_PL] [Marleza](https://osu.ppy.sh/users/5396181), ![][flag_PL] [Open](https://osu.ppy.sh/users/7082921) |
```
Comment (Joehuu) [inline wiki/Tournaments/MP4K/3/en.md]:
```suggestion
| Coordinators | ![][flag_PL] [Marleza](https://osu.ppy.sh/users/5396181), ![][flag_PL] [Open](https://osu.ppy.sh/users/7082921) |
```

**Pull request merged**